tests/app/submitter/test_converter.py

```diff
@@ -2,204 +2,329 @@
 
 import pytest
 
+from app.data_models.metadata_proxy import MetadataProxy
 from app.questionnaire.questionnaire_schema import QuestionnaireSchema
-from app.submitter.converter import DataVersionError, convert_answers
+from app.submitter.converter import convert_answers
+from app.submitter.converter_v2 import DataVersionError, convert_answers_v2
+from tests.app.submitter.conftest import get_questionnaire_store
 
 SUBMITTED_AT = datetime.now(timezone.utc)
 
 
-def test_convert_answers_flushed_flag_default_is_false(
-    fake_questionnaire_schema, fake_questionnaire_store
+@pytest.mark.parametrize(
+    "version",
+    (
+        "v1",
+        "v2",
+    ),
+)
+def test_convert_answers_v2_flushed_flag_default_is_false(
+    fake_questionnaire_schema, version
 ):
-    answer_object = convert_answers(
-        fake_questionnaire_schema, fake_questionnaire_store, {}, SUBMITTED_AT
-    )
-
-    assert not answer_object["flushed"]
+    questionnaire_store = get_questionnaire_store(version)
 
+    if version == "v2":
+        answer_object = convert_answers_v2(
```

Comment:
Like the questionnaire store get the callable conditionally, instead of duplicating the 2 calls.
i.e 
```py
converter = convert_answers_v2 if v2 else convert_answers
answer_object = converter(
            fake_questionnaire_schema, questionnaire_store, {}, SUBMITTED_AT
        )
```

Reply:
Updated 👍